rubygem-foreman_gutterball.spec

```diff
@@ -71,6 +71,10 @@ gem install --local --install-dir .%{gem_dir} --force %{SOURCE0} --no-rdoc --no-
 
 %{__install} --directory %{buildroot}%{foreman_bundlerd_dir}
 
+#create symlinks for important scripts
+mkdir -p %{buildroot}/usr/share/foreman/script/foreman-debug.d/
+ln -sv %{buildroot}/script/gutterball-debug.sh %{buildroot}/usr/share/foreman/script/foreman-debug.d/gutterball-debug.sh
```

Comment:
This might be better:

    install -Dpm 0755 script/gutterball-debug.sh %{buildroot}%{_datadir}/foreman/script/foreman-debug.d/20-gutterball-debug

This creates the directory if it does not exist and also makes sure it has executable permissions. Also scripts are executed in order, thus the number.